packages/elements/test/integration/lib/helpers.ts

```diff
@@ -1,20 +1,15 @@
 import { Page, TestInfo, expect, test } from '@playwright/test';
 import { isHeadless } from './browser';
 
-export function sleep(n = 1000) {
```

Comment:
But... without `sleep`, how will I be able to dream? 🥺

Reply:
No sleep for the wicked